test/command/5700-metadata-file-2.yml

```diff
@@ -0,0 +1 @@
+desc: Both of these files should be loaded.
```

Comment:
maybe also add a `title:` field to this file to test how the two files are merged.

Reply:
This field was added in the most recent series of commits.